group_vars/all

```diff
@@ -1,7 +1,9 @@
 ---
 #Network
 hotspot_ip: 192.168.2.1
+hotspot_source_range : 192.168.2.2-192.168.2.254
 external_hotspot_ip: 192.168.3.1
+external_hotspot_source_range: 192.168.3.2-192.168.3.254
 hotspot_interface: wlan0
 lan_interface: eth0
 client_interface: wlan1
```

Comment:
Shouldn't we also deal with some `client_external_interface: wlan2` as well?

https://github.com/ideascube/ansiblecube/blob/oneUpdateFile/roles/iptables/templates/iptables.rules.j2#L17

Reply:
I don't get which scenario can this happen?